trifle/cli/src/trifle-cli.ts

```diff
@@ -38,6 +39,45 @@ import { PublicKeyMismatchError } from "@metaplex-foundation/mpl-auction-house";
 //   chalk.green(figlet.textSync("Trifle CLI", { horizontalLayout: "full" })),
 // );
 
+const addTransferEffectsOptions = (cmd: Command) => {
```

Comment:
This is really clever and nicely done!

Reply:
Thank you @blockiosaurus! Compliments from a talented engineer are always appreciated.